src/profile.command.ts

```diff
@@ -45,6 +49,36 @@ export abstract class ProfileCommand extends Command {
         }
     }
 
+    public getAccount(options: ProfileOptions): Account {
+        const profile = this.getProfile(options);
+
+        const password = options.password || readlineSync.question('Enter your wallet password: ');
+        new PasswordValidator().validate(password);
+        const passwordObject = new Password(password);
+
+        if (!profile.isPasswordValid(passwordObject)) {
+            throw new ExpectedError('The password you provided does not match your account password');
+        }
+
+        return profile.simpleWallet.open(passwordObject);
+    }
+
+    public getAccountAndProfile(options: ProfileOptions): {account: Account, profile: Profile} {
+        const profile = this.getProfile(options);
+        const account = this.getAccount(options);
```

Comment:
I don't like the idea of calling this.getProfile() twice (one at line 67 and another at line 53).

What we could do is to put ``getAccount()`` logic inside the profile model, and rename it to ``decryptProfile``.

Reply:
added a `profile.decrypt()` method